adapters/copy.go

```diff
@@ -0,0 +1,32 @@
+package adapters
+
+import (
+	"github.com/smartcontractkit/chainlink/store"
+	"github.com/smartcontractkit/chainlink/store/models"
+	"github.com/tidwall/gjson"
+	"go.uber.org/multierr"
+)
+
+// Copy obj keys refers to which value to copy inside `data`,
+// each obj value refers to where to copy the value to inside `data`
+type Copy struct {
+	models.JSON
+}
+
+// Perform returns the copied values from the desired mapping within the `data` JSON object
+func (c *Copy) Perform(input models.RunResult, store *store.Store) models.RunResult {
+	var merr error
+	c.ForEach(func(from, to gjson.Result) bool {
+		val := input.Data.Get(from.String())
+		if val.Exists() && len(to.String()) > 0 && !to.IsObject() {
+			data, err := input.Data.Add(to.String(), val.String())
+			multierr.Append(merr, err)
```

Comment:
This is a functional method that returns the new value. So you need to reassign `err`:

```golang
err = multierr.Append(merr, err)
```